packages/core/lib/commands/init/initSource/.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
We should consider ignoring this in the webpack config. I don't think we want to copy this over for users when they do a `truffle init`, or do we?

Reply:
Yes, I mentioned this potential problem in the PR description; I agree we likely want to exclude it somehow.  The webpack config would be one way, although not necessarily the only way.  (E.g., it could also be done by modifying how the `truffle init` command works so that it doesn't copy hidden files, or by moving the file one directory up and modifying it appropriately.)  Do you think the webpack config is how it should be done?

Reply:
If we go the way of adding it to the webpack config, it looks like [this][https://eslint.org/docs/user-guide/configuring/configuration-files#using-a-configuration-file) is what we want.